voice/errors.md

```diff
@@ -9,6 +9,8 @@ Errors that occur during an active call will be delivered to the `disconnectUrl`
 * User rate limits are reached
 * System rate limits are reached
 
+Errors can also be found for a specific call by [querying the call data in the API](methods/calls/getCallsCallId.md). 
```

Comment:
The writer in me prefers the phrase "using the API" over "in the API." It's up to you if you agree or disagree, it probably doesn't matter that much. Let me know what you decide and I will approve.